### ISSUE TITLE
Tag placement in 0.13 is suboptimal for lists

### ISSUE THREAD

**Issue by nriley** [dexterleng/homerow#24]
**Describe the bug**

In 0.12, with lists (such as in sidebars), the tags were vertically aligned with the list rows or appeared as a "badge" (top right) of icons. In 0.13, they appear below the list items, making it harder to tell quickly which tag corresponds with which list item. At least in the example below, I don't feel like this is an improvement.

**Screenshots / video**

0.12:
![image](https://user-images.githubusercontent.com/68303/209996444-d0ceb85f-ef23-4d03-9538-12ececf77ef3.png)
0.13:
![image](https://user-images.githubusercontent.com/68303/209996467-15f93108-b958-4a42-8c23-93e552d2905b.png)

**Steps to reproduce the bug**

Trigger Homerow in a Cocoa app with a sidebar (example above is Shrugs).

**Comment by dexterleng** [comment]
I agree - I would like to have '>' labels for list items or table rows.

**Comment by dexterleng** [comment]
I gave horizontal labels a shot, It's not perfect - not every vertical list an accurate or present `AXOrientation` attribute value.

![CleanShot 2022-12-31 at 20 43 06](https://user-images.githubusercontent.com/34204380/210137124-8ccb7e80-6fe3-4e44-b83e-6b9fa784a094.png)


[homerow-horizontal-labels.zip](https://github.com/dexterleng/homerow/files/10327348/homerow-horizontal-labels.zip)

**Comment by nriley** [comment]
I think on balance I liked the top right position better. How about having the labels at the center right? That way if it cuts off anything it's at the end of tex rather than the beginning.

**Comment by dexterleng** [comment]
Top right or center right positioning blocks small buttons.

![](https://cdn.discordapp.com/attachments/883936912119840811/1054352397562023936/CleanShot_2022-12-19_at_18.59.182x.png)

**Comment by nriley** [comment]
I meant specifically for vertical lists, not for everything…

**Comment by dexterleng** [comment]
I tried using the horizontal labels and I found that having labels in inconsistent locations (either horizontal or vertical) is worse than just vertical labels everywhere. My eyes have to guess to either look bottom-center or middle-left.

I think the labels should be consistent in their position, and for most cases bottom-center is fine for me.

Maybe a thin, light border around wide or tall UI elements would help with this:

![CleanShot 2023-01-09 at 13 20 16@2x](https://user-images.githubusercontent.com/34204380/211245461-191e26b6-ca35-4614-a27a-8d27e5ffffa4.png)
